docs/PersonalApplications.rst

```diff
@@ -504,6 +510,103 @@ your own Wiki page. See the :ref:`Working With Wikis <Wiki>` chapter for the ful
         However, if you have :ref:`made public <MakingPublic>` for your own wiki, anyone can access, read and edit it via the link.
 
 
+.. _Managing-Fav-Apps:
+
+=======================================
+Managing your favorite applications
+=======================================
+
+Starting with eXo Platform 6, you can quickly access your favourite applications 
+through the Applications drawer.
+
+To access your favourite applications, you just need to click on this button |AC_image0| from
+the top navigation bar.
+
+|AC_image1|
+
+A drawer will open displaying:
+
+-  Mandatory applications defined by an administrator pinned in the drawer, alphabetically sorted but not draggable
+
+-  Your favorite applications alphabetically sorted, by default, could be dragged and dropped to define your own choice.
+
+|AC_image2|
+
+.. note:: A separation line is present to differentiate mandatory applications from your favorite ones.
+ 
+As cited above, you can reorganize your favorite applications by dragging and dropping their icons according to your choice.
+
+|AC_image3|
```

Comment:
The visual of this animation does not show the drag and drop effect. Can we change to something clearer ?

Reply:
I used an application to make it, I don't know if we can make it better